src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -3247,6 +3247,16 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "UniversalQRScanner_TestRestoredFromBinAlert_Message" = "Der QR-Code wurde bereits auf Ihrem Smartphone registriert. Der Test wird aus dem Papierkorb wiederhergestellt.";
 
+"UniversalQRScanner_MaxPersonAmountAlert_warningTitle" = "Warnung";
+
+"UniversalQRScanner_MaxPersonAmountAlert_errorTitle" = "Fehler";
+
+"UniversalQRScanner_MaxPersonAmountAlert_message" = "Die Corona-Warn-App darf nicht zur  Zertifikatsprüfung durch Dritte verwendet werden. Nutzen Sie dafür die CovPassCheck-App.\n\nSie können maximal für %i Personen Zertifikate hinzufügen. Danach können Sie keine neuen Zertifikate mehr für weitere Personen einscannen.\n\nWeitere Informationen zum  Impfzertifikat finden Sie in den FAQ.";
```

Comment:
In Figma, there is a small difference between the text for "Warnung" and "Fehler":
Warnung: Danach können Sie keine neuen Zertifikate mehr für weitere Personen einscannen.
Fehler: Sie können keine neuen Zertifikate mehr für weitere Personen einscannen.
Because when the halfway warning comes up, you can still scan in certificates because you have not reached the limit yet.